examples/seismic/tutorials/07_DRP_schemes.ipynb

```diff
@@ -492,23 +489,21 @@
     "# Define our custom FD coefficients:\n",
     "x, z = model.grid.dimensions\n",
     "# Upper layer\n",
-    "weights_u = np.array([ 2.00462e-03, -1.63274e-02,  7.72781e-02, \n",
-    "                      -3.15476e-01,  1.77768e+00, -3.05033e+00,  \n",
-    "                       1.77768e+00, -3.15476e-01,  7.72781e-02, \n",
-    "                      -1.63274e-02,  2.00462e-03])\n",
+    "weights_u = np.array([ 2.00462e-03, -1.63274e-02, 7.72781e-02, \n",
+    "                      -3.15476e-01, 1.77768e+00, -3.05033e+00,  \n",
+    "                       1.77768e+00, -3.15476e-01, 7.72781e-02, \n",
+    "                      -1.63274e-02, 2.00462e-03])\n",
     "# Lower layer\n",
-    "weights_l = np.array([  0.      ,  0.      ,  0.0274017, \n",
-    "                       -0.223818,  1.64875 , -2.90467,  \n",
-    "                        1.64875 , -0.223818,  0.0274017,  \n",
-    "                        0.      ,  0.       ])\n",
-    "# Create the Devito Coefficient objects:\n",
+    "weights_l = np.array([0.0274017, -0.223818, 1.64875, -2.90467,\n",
+    "                      1.64875, -0.223818, 0.0274017])\n",
+    "\n",
     "ux_u_coeffs = weights_u/x.spacing**2\n",
     "uz_u_coeffs = weights_u/z.spacing**2\n",
     "ux_l_coeffs = weights_l/x.spacing**2\n",
     "uz_l_coeffs = weights_l/z.spacing**2\n",
     "\n",
     "u_lap = u_DRP.dx2(weights=ux_u_coeffs) + u_DRP.dy2(weights=uz_u_coeffs)\n",
-    "l_lap = u_DRP.dx2(weights=ux_l_coeffs) + u_DRP.dy2(weights=uz_l_coeffs)\n",
+    "l_lap = u_DRP.dx2(weights=ux_l_coeffs, fd_order=order-4) + u_DRP.dy2(weights=uz_l_coeffs, fd_order=order-4)\n",
```

Comment:
??

Reply:
With the new interface, you can completely truncate the zeroes off the end of the stencil in the lower section of the domain. Think it should reduce computational cost unless there's some symbolic optimisation during lowering which removes the zero coefficients anyway